opendatafabric/src/lib.rs

```diff
@@ -8,6 +8,8 @@
 // by the Apache License, Version 2.0.
 
 #![feature(backtrace)]
+#![feature(provide_any)]
+#![feature(error_generic_member_access)]
```

Comment:
This is needed by `thiserror` crate to work on nightly rustc - they are starting some big refactoring of the `Error` trait.